File: calculator.ts



const start = Date.now();

let x = 1;
let y = 1;

for (let i = 0; i < 1_000; i++) {
    x *= 10;
    y += 1;
}

console.log("Final value of x: " + x);
console.log("Final value of y: " + y);

const end = Date.now();
const duration = end - start;
console.log("Execution time: " + duration + " ms");

File: calculator.py
from time import time


start = time()

x = 1
y = 1

for i in range(1_000):
    x *= 10
    y += 1


print(f"Final value of x: {x}")
print(f"Final value of y: {y}")

end = time()
duration = end - start
print(f"Execution time: {duration * 1000} ms")
